docs/how-tos/train-deploy-ml.md

```diff
@@ -435,44 +475,43 @@ In the Viam app, navigate to your list of [**DATASETS**](https://app.viam.com/da
 
 Click **Train model** and follow the prompts.
 
-Select to train a new model or update an existing model.
-You can train or update using **Built-in training** or using a [training script](/services/ml/training-scripts/) from the Viam Registry.
+You can train your model using **Built-in training** or using a [training script](/services/ml/training-scripts/) from the Viam Registry.
 
 Click **Next steps**.
 
 {{<imgproc src="/tutorials/data-management/shapes-dataset.png" resize="1200x" declaredimensions=true style="width:500px" alt="The shapes dataset." class="imgzoom fill aligncenter">}}
 
 {{% /tablestep %}}
 {{% tablestep %}}
-**2. Select the details for your ML model**
+**2. Fill in the details for your ML model**
 
-- Enter a name or use the suggested name for your new model.
-- For built-in training scripts, select a **Model Type**.
-  Depending on the training script you've chose, you may have a number of these options:
+- Enter a name for your new model.
+- For built-in trainings, select a **Task Type**:
   - **Single Label Classification**: The resulting model predicts one of the selected labels or `UNKNOWN` per image.
     Select this if you only have one label on each image. Ensure that the dataset you are training on also contains unlabeled images.
```

Comment:
It’s confusing because the app seems to use “label” in place of “tag” for classifiers. Not sure if you want to fix in this PR.

Reply:
Agreed. At this point in the UI it is called a label. So I think this is the way

Reply:
Except for when the UI calls it a tag 😅